### PR TITLE
Scrum 2860

### PR DESCRIPTION
**Pull request by ianlongden**

### CODE REVIEW

**Comment by ianlongden** [comment]
So I only have the code to get the sql and results for this and not a complete ticket but as there are fixing to the docker files it might be worth merging into main anyway?

File: docker/api.dockerfile [hunk 1]
```diff
@@ -9,6 +9,10 @@ WORKDIR /usr/local/bin/src/literature
 
 ADD . .
 
-RUN pip install -r requirements.txt
+RUN pip3 install --upgrade pip
```
Comment (valearna) [inline docker/api.dockerfile:12]: You mentioned why you need to do this at standup but I didn't understand what the reason was

Comment (valearna) [reply docker/api.dockerfile:12]: Oh, I see the comments on scrum-blueteam. Everything makes sense

**Review by valearna — APPROVED** [review]

**Comment by ianlongden** [comment]
Yes, I wrote on several things so not surprised it was missed☺️

**Pull request merged**